exampleCourse/questions/demo/workspace/jupyterlab/workspace/Workbook.ipynb

```diff
@@ -1,14 +1,26 @@
 {
  "cells": [
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fibonacci numbers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Fibonacci numbers are 1, 1, 2, 3, 5, 8, ..., where each number is the sum of the two before it.\n",
```

Comment:
```suggestion
    "The Fibonacci numbers are 1, 1, 2, 3, 5, 8, ..., where each number is the sum of the two before it. These are numbered $F_1, F_2, ...$\n",
```

It's always bothered me that this question doesn't specify whether the sequence starts from `n=0` or `n=1` 😄 (the grader assumes that it starts from `n=1`, I believe).

Reply:
Do you want me to make the change in the vs question as well?

Reply:
Should we also change the two other copies of this question as well? :slightly_smiling_face:

Reply:
Yes please! It would be great to fix this everywhere.

Reply:
Where "everywhere" = "all files in this question and all questions that refer to the Fibonacci sequence" 😄 -- `demo/autograder/codeEditor`, `demo/autograder/codeUpload`, `demo/manualGrade/codeUpload`.

Reply:
oh... missed that. I will change it